prow/plugins.yaml

```diff
@@ -160,6 +160,18 @@ config_updater:
       name: plugins
 
 plugins:
+  bazelbuild/rules_k8s:
+  - approve  # Allow OWNERS to /approve
+  - assign  # Allow /assign and /cc
+  - blunderbuss  # Auto-assign people
+  - help  # Support /help and /good-first-issue
+  - hold  # Support /hold to delay merge
+  - lgtm  # Allow /lgtm
+  - lifecycle  # Allow /lifecycle stale
+  - size  # Auto-label size of PR
+  - trigger  # Allow people to configure CI jobs to /test
```

Comment:
define trusted org at top, and do they have a prow job?

Reply:
Done.

No prow job yet, will work on that next